service/Abstractions/Pipeline/MimeTypes.cs

```diff
@@ -11,9 +11,9 @@ public static class MimeTypes
     public const string PlainText = "text/plain";
     public const string MarkDown = "text/plain-markdown";
     public const string Html = "text/html";
-    public const string MsWord = "application/msword";
-    public const string MsPowerPoint = "application/mspowerpoint";
-    public const string MsExcel = "application/msexcel";
+    public const string MsWord = "application/vnd.openxmlformats-officedocument.wordprocessingml.document";
```

Comment:
at some point we're going to fix the support for old type, could you leave this and introduce a new one for modern Word? Something like 
```
MsWord = "application/msword";
MsWordOpenXML = "application/vnd.openxmlformats-officedocument.wordprocessingml.document";
```

I would apply the same suffix also to the other 2 below -- or leave them untouched until future work that will support both old and new Excel/Powerpoint formats, ie limit the PR scope

Reply:
It is OK to use the X suffix as the old code used it? Also, should I apply the suffix in the decoders as well as seen in this code?

```cs
case MimeTypes.MsWordX:
    this._log.LogDebug("Extracting text from MS Word file {0}", uploadedFile.Name);
    content = new MsWordDecoder().ExtractContent(fileContent);
    break;
```

Reply:
sure these constants are internal only. 

About the decoder, if you mean TextExtractionHandler yes, I would change `case MimeTypes.MsWord:` to `case MimeTypes.MsWordX:`, so ".doc" files will be caught by the default case and log a warning. 

You could also add a case for `case MimeTypes.MsWord:` and log a more explicit warning, something like `Legacy Word 97 format not supported, it is recommended to migrate to the newer OpenXML format`.